crates/cargo-wdk/tests/new_command_test.rs

```diff
@@ -207,11 +200,23 @@ fn create_and_build_new_driver_project(driver_type: &str) -> (String, String) {
     cmd.args(["build"]).current_dir(&driver_path);
 
     let cmd_assertion = cmd.assert().failure();
-    tmp_dir
-        .close()
-        .expect("Unable to close temp dir after test");
```

Comment:
Removed because I would have had to duplicate it in the above newly added early return as well. It isn't needed anyway as `drop` takes care of closing